components/css/newsletter-text-only/_contentPage.scss

```diff
@@ -2,172 +2,194 @@ $white:#ffffff;
 $black:#000000;
 $light-gray:#aaaaaa;
 $dark-gray:#2e2e2e;
-$gold:#c28e0e;
+$campusGold:#c28e0e;
 
 .intro {
-	background-color: #c28e0e;
+	background-color: $campusGold;
 	padding: 1.5em;
-
-
-	&__title {	
-	font-size: 2em;
-	line-height: 1.3em;
-	color: $black;
-	text-align: center;
-	text-transform:uppercase;
-	padding: .5em 0;
-	}
 	
-	&__issue {
-	font-size: 1em;
-	line-height: 1em;
-	color: $black;
-	text-align: center;
-	padding: 0;
-	}
+		&__title {	
+			font-size:calc-toRem(32);
+			line-height: 1.3em;
+			color: $black;
+			text-align: center;
+			text-transform:uppercase;
+			padding: .5em 0;
+		}
 	
-	&__issue::after {
-	content:" ";
-	display: block;
-	width: 30%;
-	height: 3px;
-	margin: .3em auto !important;
-	background-color: $white;	
-	}
+		&__issue {
+			font-size: 1em;
+			line-height: 1em;
+			color: $black;
+			text-align: center;
+			padding: 0;
+		}
+	
+		&__issue::after {
+			content:" ";
+			display: block;
+			width: 30%;
```

Comment:
I talked with Garrett and here wants to hardcode the width of these instead of using a percentage. Let's see how 120px looks.